activemodel/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+*   Validate multiple contexts on `valid?` and `invalid?` at once.
+
+    Example:
+    
+        class Person
+          include ActiveModel::Validations
+    
+          attr_reader :name, :title
+          validates_presence_of :name, on: :create
+          validates_presence_of :title, on: :update
+        end
+    
+        person = Person.new
+        person.valid?([:create, :update])    # => true
```

Comment:
How many changes would be necessary to make `person.valid?(:create, :update)` work?